monors.py

```diff
@@ -217,6 +217,56 @@ def load_statuses (self):
 
         return statuses
 
+    def try_report_failures (self):
+        success = self.is_successful (statuses)
+        if success is not None:
+            comment = ""
+
+            if success is True:
+                message = "Required status checks are successful."
+                logging.info (message)
+
+                comment += message + "\n"
+            else:
+                message = "Required status checks are not successful:"
+                logging.info (message)
+
+                comment += message + "\n"
+
+                for context in self.mandatory_context:
+                    status_text = statuses [context].state if context in statuses else "pending"
+
+                    message = " - \"%s\" state is \"%s\"" % (context, status_text)
+                    logging.info (message)
+
+                    if status_text != "success":
+                        comment += message + "\n"
+
+            last_comment = None
+            for c in self.comments:
+                if c.login != self.cfg ["user"].encode ("utf8"):
+                    continue
+                if not c.body.startswith ("Required status checks are not successful:".encode ("utf8")) and not c.body.startswith ("Required status checks are successful.".encode ("utf8"))):
```

Comment:
I guess we can simplify this to check
```suggestion
                if not c.body.startswith ("Required status checks are ".encode ("utf8")):
```